src/helperFunctions/yara_binary_search.py

```diff
@@ -81,3 +97,15 @@ def get_yara_error(rules_file):
         return None
     except Exception as e:
         return e
+
+
+class YaraBinarySearchScannerDbInterface(MongoInterfaceCommon):
+
+    READ_ONLY = True
+
+    def get_file_paths_of_files_included_in_fo(self, fo_uid: str) -> List[str]:
+        fs_organizer = FS_Organizer(self.config)
+        return [
```

Comment:
May be nitpicking, but since this can be a huge list (thousands of paths) maybe use an iterator instead (round instead square brackets)